cellbase-app/app/azure/arm/webservers/azuredeploy.json

```diff
@@ -0,0 +1,432 @@
+                "description": "Admin password on all VMs."
+            }
+        },
+        "LogAnalyticsWorkspaceID": {
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace ID"
+            }
+        },
+        "logAnalyticsWorkSpaceKey":{
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace Key"
+            }
+        },
+        "LogAnalyticsWorkspaceResourceID": {
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace Resource ID"
+            }
+        }
+    },
+    "variables": {
+        "vmssName":"cellbasevmss",
+        "namingPrefix":"webservers",
+        "sshKeyPath": "[concat('/home/',parameters('adminUsername'),'/.ssh/authorized_keys')]",
+        "loadBalancerName": "[concat(variables('namingPrefix'), '-lb')]",
+        "lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName'))]",
+        "natPoolName": "[concat(variables('namingPrefix'), 'natpool')]",
+        "natStartPort": 50000,
+        "natEndPort": 50119,
+        "natBackendPort": 22,
+       "privateFrontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/privateFrontEnd')]",
+        "imageReference": {
+            "publisher": "Canonical",
+            "offer": "UbuntuServer",
+            "sku": "18.04-LTS",
+            "version": "latest"
+        },
+        "logAnalyticsAgent": "[concat('docker run --privileged -d -v /var/run/docker.sock:/var/run/docker.sock -v /var/log:/var/log -v /var/lib/docker/containers:/var/lib/docker/containers -e WSID=',parameters('logAnalyticsWorkSpaceID'),' -e KEY=',parameters('logAnalyticsWorkSpaceKey'),' -p 127.0.0.1:25225:25225 -p 127.0.0.1:25224:25224/udp --name=omsagent -h=`hostname` --restart=always microsoft/oms')]",
+    
+        "_comment": "The cloud init below is used to install docker then run the cellbase container",
+        "cloud-init": [
+            "#cloud-config",
+            "package_upgrade: true",
+            "package_reboot_if_required: true",
+            "runcmd:",
+            "- curl -fsSL https://get.docker.com/ | sh",
+            "- curl -fsSL https://get.docker.com/gpg | sudo apt-key add -",
+            "[concat('- ', variables('logAnalyticsAgent'))]",
+            "[concat('- docker run --name=cellbase -d -p 8080:8080 -p 8443:8443 --restart=always -e CELLBASE_DATABASES_MONGODB_HOST=',string(parameters('mongoHosts')),' -e  CELLBASE_DATABASES_MONGODB_USER=',string(parameters('mongoUsername')),' -e  CELLBASE_DATABASES_MONGODB_PASSWORD=',string(parameters('mongoPassword')),'  -e CELLBASE_DATABASES_MONGODB_OPTIONS_AUTHENTICATION_DATABASE=admin  -e CELLBASE_DATABASES_MONGODB_OPTIONS_REPLICA_SET=  -e  CELLBASE_DATABASES_MONGODB_OPTIONS_READ_PREFERENCE=nearest  -e CELLBASE_DATABASES_MONGODB_OPTIONS_SSL_ENABLED=true  ', parameters('cellbaseContainerImage'))]",
```

Comment:
nit: could split this onto multiple lines if you wanted like I've done in the Daemonvm